src/runtime/internal/vite/dispatcher.mjs

```diff
@@ -8,5 +8,16 @@ export default defineHandler(async (event) => {
   }
   return fetch(new URL(`/${m.params?._ || ""}${event.url.search}`, event.url), {
     viteEnv: m.data.service,
+    method: event.req.method,
+    headers: event.req.headers,
```

Comment:
Note for later: We might need to clone headers properly (HOST in particular should be overriden and we need to add x-forwarded headers)